src/main/java/com/minecolonies/entity/ai/citizen/miner/EntityAIStructureMiner.java

```diff
@@ -444,16 +448,16 @@ private boolean buildNextBlockInShaft()
         BlockPos ladderPos  = getOwnBuilding().getLadderLocation();
         int      lastLadder = getLastLadder(ladderPos) + 1;
 
-        int xOffset = 3 * getOwnBuilding().getVectorX();
-        int zOffset = 3 * getOwnBuilding().getVectorZ();
+        int xOffset = SHAFT_RADIUS * getOwnBuilding().getVectorX();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'xOffset' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)